### PR TITLE
use name instead of slugline for templates data based on new config [SDBELGA-804]

### PR DESCRIPTION
**Pull request by devketanpro**
https://github.com/superdesk/superdesk-belga/pull/476

### CODE REVIEW

File: server/planning/assignments/assignments.py [hunk 1]
```diff
@@ -381,6 +381,10 @@ def send_assignment_notification(self, updates, original=None, force=False):
         coverage_type = updates.get("planning", original.get("planning", {})).get("g2_content_type", "")
         slugline = updates.get("planning", original.get("planning", {})).get("slugline", "with no slugline")
 
+        subject_field = app.config.get("ASSIGNMENT_MAIL_SUBJECT_FIELD")
+        if subject_field:
+            slugline = updates.get(subject_field, original.get(subject_field, slugline))
```
[the remark this answers is not in the document]
Comment (devketanpro) [reply server/planning/assignments/assignments.py:386]: yep, I'll add it, Thanks

File: server/planning/planning_notifications.py [hunk 2]
```diff
@@ -304,7 +304,7 @@ def _send_user_email(user_id, contact_id, source, meta_message, data):
 
     admins = app.config["ADMINS"]
 
-    data["subject"] = "Superdesk assignment" + ": {}".format(data.get("slugline") if data.get("slugline") else "")
+    data["subject"] = render_template("assignment_mail_subject" + ".txt", **data)
```
Comment (petrjasek) [inline server/planning/planning_notifications.py:307]: not sure why this string concatenation, you can just do `"assignment_mail_subject.txt"`

File: server/planning/templates/assignment_mail_subject.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Superdesk assignment planning: {{slugline}}
```
Comment (petrjasek) [inline server/planning/templates/assignment_mail_subject.txt:1]: I think it should be just `Superdesk assignment: {{slugline}}` no need to change it

Comment (devketanpro) [reply server/planning/templates/assignment_mail_subject.txt]: my bad

File: server/planning/templates/assignment_mail_subject.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Superdesk assignment : {{slugline}}
```
Comment (petrjasek) [inline server/planning/templates/assignment_mail_subject.txt:1]: without the space before ` :`

**Pull request merged**